src/components/organisms/BottomNav.js

```diff
@@ -25,6 +26,13 @@ const StyledBottomNavigation = styled(BottomNavigation)`
   .MuiBottomNavigationAction-label.Mui-selected {
     font-size: 12px;
   }
+  ${bps.down("md")} {
```

Comment:
@nandiheath 
This actually changes the alignment of the bottom nav a little bit.
![image](https://user-images.githubusercontent.com/5630513/73606626-3eb48000-45e7-11ea-8fda-8b984be79dc4.png)